dev/reference/mysql-compatibility.md

```diff
@@ -149,3 +149,23 @@ TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
         - For MySQL 5.7: `OFF`
         - For MySQL 8.0: `ON`
     
+### Date and Time
+
+#### Time zone
+
+By default, MySQL time zone is set to the server's time zone and calculated with the system time zone rules such as the beginning of daylight saving time. Befor loading [time zone information table](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation), you cannot specify the time zone with the time zone name. 
+
+All the time zone names are available in TiDB without loading the time zone information table. The system calculates based on its own time zone rules which is generally the `tzdata` package. The calculation rules cannot be modified by loading time zone information table. 
+
+> **Warning**:
+>
+> TiKV calculates the time-related expressions received with its built-in time zone calculation rules instead of the system time zone rules. If these two kinds of time zone rules do not match with each other, it may occur that the inserted time data cannot be read again. In addition, if you install the tzdata 2018a time zone rules, when you set the time zone to Asia/Shanghai or to the server's time zone which is Asia/Shanghia, TiDB 3.0 RC.1 can be inserted to `1988-04-17 02:00:00`. But this record can not be read again by certain types of SQL statements. The reason is that the time does not exist in Asia/Shanghai time zone according to the tzdata 2018i rules on which the TiKV 3.0 based. (For example, the daylight saving time is shifted back by an hour.)
```

Comment:
`(For example, the daylight saving time is shifted back by an hour.)` -> `(That is, the time does not exist because the DST shifts forward time by an hour.)`